File: BE/src/main/java/com/lisade/togeduck/dto/response/RouteDetailResponse.java
package com.lisade.togeduck.dto.response;

import com.fasterxml.jackson.annotation.JsonFormat;
import java.time.LocalDateTime;
import java.time.LocalTime;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class RouteDetailResponse {

    Long id;
    @JsonFormat(pattern = "yyyy.MM.dd HH:mm")
    LocalDateTime startedAt;
    String source;
    String destination;
    @JsonFormat(pattern = "HH:mm")
    LocalTime expectedAt;
    @JsonFormat(pattern = "HH:mm")
    LocalTime arrivalAt;
    Integer totalSeats;
    Integer reservedSeats;
    Integer cost;
}

File: AOS/app/src/main/java/com/softeer/togeduck/data/dto/response/article_detail/RouteDetailResponse.kt
package com.softeer.togeduck.data.dto.response.article_detail

data class RouteDetailResponse(
    val arrivalAt: String,
    val cost: Int,
    val destination: String,
    val expectedAt: String,
    val id: Int,
    val reservedSeats: Int,
    val source: String,
    val startedAt: String,
    val totalSeats: Int
)